apps/mobile/src/features/send/send-sheet.tsx

```diff
@@ -28,14 +30,17 @@ export function SendSheet() {
       onAnimate={handleAnimatedPositionChange}
       onDismiss={handleDismiss}
     >
-      <Send accountId={accountId} />
+      <Send accountId={accountId} asset={asset} />
     </FullHeightSheet>
   );
 }
 
 function useInitialSendParams() {
   const params = useGlobalSearchParams();
   const accountId = isString(params.accountId) ? params.accountId : undefined;
+  const tokenId = isString(params.tokenId) ? params.tokenId : undefined;
+  // FIXME LEA-3125: we need to refactor this so we can also send SIP-10 tokens
```

Comment:
Now we can send SIP-10 I need to improve this. I'll do it in a follow up and update some of the SIP-10 services as I go to make it easier to work with specific tokens.